lib/cc/engine/scalastyle.rb

```diff
@@ -48,7 +48,7 @@ def run
       private
 
       def project_files
-        Dir.glob("**/*scala")
+        @engine_config['include_paths'] || []
```

Comment:
For this engine, is passing `[]` the same as `["./"]`? It's not a requirement of the spec, but I've found it useful when developing engines to have them analyze the whole project directory when `include_paths` aren't given.